invenio_sequencegenerator/__init__.py

```diff
@@ -23,3 +23,10 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 """Invenio module for generating sequences."""
+
+from __future__ import absolute_import, print_function
+
+from .version import __version__
+
+__all__ = ('__version__',
+           )
```

Comment:
You don't have to break the line here. If you prefer to break it, then in this case it is preferable following style:

```python
__all__ = (
    '__version__',
)
```